### ISSUE TITLE
[Bug]: Datapack is not applied to dirt path / stone bricks

### ISSUE THREAD

**Issue by No3371** [Fuzss/blockrunner#16]
### Mod Loader (Required)

Fabric

### Minecraft Version (Required)

1.20.1

### Mod Version (Required)

8.0.2

### Notes (Required)

Unless `minecraft:dirt_path`/`#minecraft_stone_bricks` is specified in the config files, the blocks use the predefined values (1.35/1.1) even if it's tagged with datapacks or with an empty config file.

There are logs related to BlockRunner but I think it's an old internal issue and not related:

<details>
```
[23:12:47] [Render thread/INFO]: Constructing common components for blockrunner:main
[23:12:47] [Render thread/INFO]: Loading client config for blockrunner
[23:12:48] [Render thread/INFO]: [STDERR]: org.objectweb.asm.tree.analysis.AnalyzerException: Error at instruction 49: Insufficient maximum stack size.
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:279)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.util.Locals.generateLocalVariableTable(Locals.java:859)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.util.Locals.getGeneratedLocalVariableTable(Locals.java:828)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.util.Locals.getLocalVariableAt(Locals.java:782)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.util.Locals.getLocalVariableAt(Locals.java:752)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.util.Locals.getLocalsAt_0_9_2(Locals.java:696)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.util.Locals.getLocalsAt(Locals.java:338)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.injection.modify.LocalVariableDiscriminator$Context.initLocals(LocalVariableDiscriminator.java:154)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.injection.modify.LocalVariableDiscriminator$Context.<init>(LocalVariableDiscriminator.java:148)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.injection.modify.ModifyVariableInjector$Context.<init>(ModifyVariableInjector.java:71)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.injection.modify.ModifyVariableInjector.preInject(ModifyVariableInjector.java:159)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.injection.code.Injector.preInject(Injector.java:258)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.preInject(InjectionInfo.java:436)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1370)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1062)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:402)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:327)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.Class.forName0(Native Method)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.Class.forName(Class.java:375)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at jdk.proxy2/jdk.proxy2.$Proxy134.<clinit>(Unknown Source)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1053)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1039)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.event.v1.core.FabricEventFactory.lambda$createResult$1(FabricEventFactory.java:50)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.update(ArrayBackedEvent.java:54)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.fabric.impl.base.event.ArrayBackedEvent.<init>(ArrayBackedEvent.java:50)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.fabric.impl.base.event.EventFactoryImpl.createArrayBacked(EventFactoryImpl.java:47)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.fabric.api.event.EventFactory.createArrayBacked(EventFactory.java:44)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.event.v1.core.FabricEventFactory.createResult(FabricEventFactory.java:50)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.client.event.v1.FabricClientEvents.<clinit>(FabricClientEvents.java:77)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.client.event.FabricClientEventInvokers.register(FabricClientEventInvokers.java:101)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.event.FabricEventInvokerRegistryImpl.register(FabricEventInvokerRegistryImpl.java:364)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.core.FabricFactories.registerEventInvokers(FabricFactories.java:51)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.event.core.EventInvokerImpl.<clinit>(EventInvokerImpl.java:17)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.event.v1.core.EventInvoker.lambda$lookup$0(EventInvoker.java:41)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:183)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.event.v1.core.EventInvoker.lambda$lookup$1(EventInvoker.java:42)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.event.v1.core.EventInvoker.register(EventInvoker.java:53)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.blockrunner.BlockRunner.registerHandlers(BlockRunner.java:36)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.blockrunner.BlockRunner.onConstructMod(BlockRunner.java:28)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.core.FabricModConstructor.registerHandlers(FabricModConstructor.java:38)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.core.FabricModConstructor.construct(FabricModConstructor.java:23)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.impl.core.FabricFactories.constructMod(FabricFactories.java:21)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.puzzleslib.api.core.v1.ModConstructor.construct(ModConstructor.java:48)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at fuzs.blockrunner.BlockRunnerFabric.onInitialize(BlockRunnerFabric.java:10)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:383)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.<init>(class_310.java:458)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:211)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
[23:12:48] [Render thread/INFO]: [STDERR]: Caused by: java.lang.IndexOutOfBoundsException: Insufficient maximum stack size.
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.objectweb.asm.tree.analysis.Frame.push(Frame.java:273)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.objectweb.asm.tree.analysis.Frame.execute(Frame.java:326)
[23:12:48] [Render thread/INFO]: [STDERR]: 	at org.objectweb.asm.tree.analysis.Analyzer.analyze(Analyzer.java:171)
[23:12:48] [Render thread/INFO]: [STDERR]: 	... 65 more
```
</details>

**Comment by Aceplante2** [comment]
Just ran into this

**Comment by Fuzss** [comment]
Fixed for 1.20.1+.
